packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -575,6 +565,13 @@ <h2 data-translate="pci_database_add_step7_title"></h2>
                         ></span>
                     </div>
                 </oui-button>
+                <div class="mt-2">
+                    <a
+                        href=""
+                        data-translate="pci_database_add_show_command"
+                        data-ng-click="$ctrl.trackAndGoToCommand()"
+                    ></a>
```

Comment:
I think that empty href are invalid. Maybe a button with `oui-link` class could be used here:
```
<button
    type="button"
    class="oui-link"
    data-ng-click="$ctrl.trackAndGoToCommand()"
>
    <span
        data-translate="pci_database_add_show_command"
    ></span>
</button>
```

Reply:
Fixed :)